java-numbers-3/pom.xml

```diff
@@ -0,0 +1,28 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>java-numbers-3</artifactId>
```

Comment:
Need to reference this module in the root pom.xml so that it gets built automatically. Look for occurrences of the other java-numbers modules and add this one in the same places.